backend/routes/auth_route.py

```diff
@@ -58,7 +60,7 @@ def login():
 
     if user and verify_password_hash(data.get('password'), user.password_salt, user.password_hash):
         access_token = create_access_token(identity=user)
-        return jsonify(access_token=access_token, userId=str(user.id), message='Login successful!'), 200
+        return jsonify(access_token=access_token, userId=str(user.id), message='Login successful!', is2fa = user.is_two_factor_auth_enabled), 200
```

Comment:
This method returns 2fa is turned on or not, but it also returns user data and access token, so it doesn't really care if there is a 2fa authorization or not 👀